.env

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_BASE_API_URL=https://survey-api.nimblehq.co/api/v1/
```

Comment:
Why did we publish these values? 🤔 
They should be set from the server instead.
If we reveal them like this, we may be spammed

Reply:
Good catch 👍 I added the PR here https://github.com/hanam1ni/react-surveyor/pull/66